src/libraries/System.Security.AccessControl/src/System/Security/AccessControl/Privilege.cs

```diff
@@ -198,85 +187,78 @@ public TlsContents()
 
                 try
                 {
-                    // Make the sequence non-interruptible
-                }
-                finally
-                {
-                    try
+                    //
+                    // Open the thread token; if there is no thread token, get one from
+                    // the process token by impersonating self.
+                    //
+
+                    SafeTokenHandle? threadHandleBefore = this.threadHandle;
+                    error = FCall.OpenThreadToken(
```

Comment:
```suggestion
                    error = Win32.OpenThreadToken(
```
and also delete FCall as alias for Win32 in this file. This has nothing to do with FCalls.